code/script.js

```diff
@@ -36,12 +51,131 @@ const showMessage = (message, sender) => {
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
+  showMessage("Hi, I'm BeautyBot, What's your name?", 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
+const welcomeMessage = (userName) => {
+  //Take the username
+
+  if (userName == '') {
+    //If text input is blank, the showMessage will appear to ask user to write their name
+    showMessage(`Name must be filled out!`, 'bot')
+    return false;
+  } else {
+    //If user types their name the message below will appear.
+    showMessage(`Hello ${userName}, welcome! Here at Technigo Beauty we love making our customers feel amazing. Let's dig into the world of beauty!`, 'bot')
+  }
+  setTimeout(() => beautyOptions(userName), 1000);
+}
+
+const beautyOptions = (userName) => {
+  //this asks the user to pick a treatment. The users name is also included in the message
+  showMessage(`Now ${userName}, which beautytreatment would you like today?✨`, 'bot')
+  //these ar choice-buttons
+  inputWrapper.innerHTML = ` 
+    <button id="nailsBtn">Nails</button>
+    <button id="hairBtn">Hair</button>
+    <button id="lashesBtn">Lashes</button>
+  `;
+
+  document.getElementById('nailsBtn').addEventListener('click', nailsChoice)
+  document.getElementById('hairBtn').addEventListener('click', hairChoice)
+  document.getElementById('lashesBtn').addEventListener('click', lashesChoice)
+
+}
+
+
+// Define your choice functions
+const nailsChoice = () => {
+  showMessage("I would like to get my nails done", 'user');
+  setTimeout(() => pickNails(), 2000)
+}
+
+const hairChoice = () => {
+  showMessage("I would like to get my hair done", 'user');
+  setTimeout(() => pickHair(), 2000)
+}
+
+const lashesChoice = () => {
+  showMessage("I would like to get my lashes done", 'user');
+  setTimeout(() => pickLashes(), 2000)
+}
+
+const pickNails = () => {
```

Comment:
this is just an extra but maybe one question more what kind of treatment, but that´s just an extra thing